components/script/dom/webglrenderingcontext.rs

```diff
@@ -1039,6 +1039,147 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
     }
 
     // https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.10
+    fn Uniform2fv(&self,
+                  _cx: *mut JSContext,
+                  uniform: Option<&WebGLUniformLocation>,
+                  data: Option<*mut JSObject>) {
+        let data = match data {
```

Comment:
Preferably fixing it here, but if you don't have the time or anything feel free to file an issue :)